app/routers/calendar.py

```diff
@@ -1,32 +1,38 @@
 from http import HTTPStatus
 
-from fastapi import APIRouter, Request
+
+from app.dependencies import get_db, templates
+from app.internal import load_parasha as lp
+from app.routers import calendar_grid as cg
+from fastapi import APIRouter, Request, Depends
 from fastapi.responses import HTMLResponse
 from starlette.responses import Response
 
-from app.dependencies import templates
-from app.routers import calendar_grid as cg
 
 router = APIRouter(
     prefix="/calendar/month",
     tags=["calendar"],
     responses={404: {"description": "Not found"}},
 )
 
+
 ADD_DAYS_ON_SCROLL: int = 42
 
 
 @router.get("/")
-async def calendar(request: Request) -> Response:
+async def calendar(request: Request, db_session=Depends(get_db)) -> Response:
```

Comment:
Please use the function that adds event to the user's calendar instead.

Reply:
which one?

Reply:
I'm sure you can find it :) Part of the work is the integration with the rest of the team

Reply:
What's the problem with using the existing function? This week's parashot are global and do not change per user

Reply:
Some people might want to see these events, and some might not. To make the event show on the calendar GUI, we should add the event to the users' calendar.

Reply:
do you want it to stay above the calendar  or in the day view?

Reply:
ok i will search it

Reply:
i change it totally.its ok?